### PR TITLE
Implemented `RpmMetadata.Append`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #388 
Implemented `RpmMetadata.Append` and added test.

### CODE REVIEW

File: src/test/java/com/artipie/rpm/RpmMetadataAppendTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+/**
+ * Test for {@link RpmMetadata.Append}.
+ * @since 1.5
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+class RpmMetadataAppendTest {
+
+    @Test
+    void appendsRecords() throws IOException {
```
Comment (genryxy) [inline src/test/java/com/artipie/rpm/RpmMetadataAppendTest.java:46]: @olenagerasimova thanks. Is it possible to add packages when `primary.xml.example` and `other.xml.example` are empty or absent? If it is so, I'd suggest to add such test case

Comment (olenagerasimova) [reply src/test/java/com/artipie/rpm/RpmMetadataAppendTest.java:46]: @genryxy now - it's not possible, there will be an invalid xml exception or smth like that. But that's a good question, I will discuss such scenario with our colleagues and make the changes accordingly.

**Review by genryxy — APPROVED** [review]
@olenagerasimova got it

**Comment by olenagerasimova** [comment]
@rultor release, tag=`1.5`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`1.5`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`1.5`

**Pull request merged**